test/GitcoinGovernor.t.sol

```diff
@@ -327,7 +336,7 @@ abstract contract AlphaGovernorPostProposalTest is ProposalTestHelper {
     bool isGovernorAlphaAdmin
   ) internal {
     // Submit the new proposal
-    vm.prank(PROPOSER);
+    vm.prank(0x4Be88f63f919324210ea3A2cCAD4ff0734425F91);
```

Comment:
Nit, probably good to have another comment specifying this address is kevinolsen.eth

Reply:
maybe pull this out into `address kevinOlsen = 0x4Be88f63f919324210ea3A2cCAD4ff0734425F91`?

Reply:
Actually this change was not necessary, reverted to `PROPOSER` in bdf9c667a72a2d4845284f04b6fbf00c8b458c28